src/createWebpackLessPlugin.js

```diff
@@ -18,7 +18,7 @@ const matchMalformedModuleFilename = /(~[^/\\]+)\.less$/;
 // This somewhat changed in Less 3.x. Now the file name comes without the
 // automatically added extension whereas the extension is passed in as `options.ext`.
 // So, if the file name matches this regexp, we simply ignore the proposed extension.
-const isModuleName = /^~[^/\\]+$/;
+const isModuleName = /^~(|@[a-z0-9-~][a-z0-9_.-~]\/)[^/\\]+$/;
```

Comment:
why change on a-z0-9_.-~?

Reply:
Oh sorry for missing '*' after `[a-z0-9_.-~]`.

And why [a-z0-9_.-~]:

> All npm packages have a name. Some package names also have a scope. A scope follows the usual rules for package names (URL-safe characters, no leading dots or underscores). When used in package names, scopes are preceded by an @ symbol and followed by a slash, e.g.
> ```
> @somescope/somepackagename
> ```

As [npm-scope](https://docs.npmjs.com/misc/scope) says, a scope follows the usual rules for package names (URL-safe characters, no leading dots or underscores). 

And [the JSON schema of package.json](http://json.schemastore.org/package) also uses the pattern:

```json
{
    "properties": {
        "name": {
          "description": "The name of the package.",
          "type": "string",
          "maxLength": 214,
          "minLength": 1,
          "pattern": "^(?:@[a-z0-9-~][a-z0-9-._~]*/)?[a-z0-9-~][a-z0-9-._~]*$"
        }
    }
}
```